source/scripts/model/Adventurer.js

```diff
@@ -51,6 +51,11 @@ class Adventurer {
         
         this.position.x += movement.x
         this.position.y += movement.y
+        
+        if(this.game != undefined
+        && this.game.camera != undefined) {
+            this.game.camera.center(this.position)
+        }
```

Comment:
Whenever accessing the game object, you don't know what's inside it, so be careful to check, so we don't get null errors.